packages/index.bzl

```diff
@@ -23,6 +23,7 @@ NPM_PACKAGES = ["@bazel/%s" % pkg for pkg in [
     "labs",
     "protractor",
     "rollup",
+    "runfiles",
```

Comment:
would you add it to .commitlint.js as well so you can feat(runfiles) ?

Reply:
:+1: Good catch.